src/_data/capability-matrix.yml

```diff
@@ -89,7 +102,10 @@ categories:
             l1: 'Yes'
             l2: fully supported
             l3: "Using Spark's <tt>combineByKey</tt> and <tt>aggregate</tt> functions."
-
+          - class: apex
+            l1: 'Yes'
+            l2: 'fully supported'
+            l3: "Default Beam translation. Currently no efficient pre-aggregation, runner should use underlying Apex functionality for this."
```

Comment:
Is there a JIRA for efficient pre-aggregation in the ApexRunner? I'd link to it here and omit "runner should use underlying Apex functionality for this", since that is a discussion for the JIRA.